cmd/agent/common/system_probe_config_helper.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build process
```

Comment:
On windows we will have an error on compile time as two `SetupSystemProbeConfig` are defined  (here and in system_probe_config_helper_unsupported.go). I think we need to require both process and not windows for this file (`// +build process,!windows`).

Reply:
Thanks for the note - I... had been playing around with this all morning and made the simple mistake of not realizing there couldn't be a space between the `,` and `!windows` so I thought this wouldn't work. 

This should hopefully now be set!